metrics/integration.go

```diff
@@ -100,6 +108,10 @@ func RegisterNewIntegrationPipelineRun(snapshotCreatedTime metav1.Time, pipeline
 	RegisterPipelineRunStarted(snapshotCreatedTime, pipelineRunStartTime)
 }
 
+func RegisterReleaseLatency(integrationTestsFinishTime metav1.Time, releaseCreationTime metav1.Time) {
```

Comment:
I may miss something, but where is this function used?

Reply:
error on my part